tests/appsec/iast/_test_iast_fixtures.py

```diff
@@ -94,9 +94,14 @@ def setup_insecure(self):
         if self.__class__.insecure_request is None:
             assert self.insecure_endpoint is not None, f"{self}.insecure_endpoint must not be None"
 
-            self.__class__.insecure_request = weblog.request(
-                method=self.http_method, path=self.insecure_endpoint, data=self.data, headers=self.headers
-            )
+            if self.http_method == "GET":
```

Comment:
Could rather add a `params=None` property to `BaseSinkTestWithoutTelemetry`. Otherwise, it will be impossible to send a request with both data (body) and params (URL parameters).

And it will also avoid a hidden logic.

Reply:
Ok, right away